bin/install.sh

```diff
@@ -30,7 +30,7 @@ function install_crds() {
 # Cleanup all namespaces
 
 function cleanup() {
```

Comment:
Why ? What's the use case ?

Reply:
We always use the same cluster to repeatedly install and update istio. That's why we need a way to reset the cluster state.

Reply:
We should separate test env scripts/configs from the 'install.sh' script. 
Maybe use the makefile to cleanup ?

Also - would it be possible to use the PR number or commit to name the environment ? 
That's the point of multi-environment, and most tests should be isolated (and if not we 
have a bug ). Same for fortio/bookinfo - they don't need to be installed in default ns, 
but some ns derived from the PR/commit.

Reply:
BTW - few benefits are that you can use a cluster for multiple PRs, and we also verify that 
Isolation works and that indeed the new installer supports multiple versions of Istio at the 
same time, which is one of the main features. 

The namespaces can be cleaned at stop, or maybe with a cron job - for example we could
delete if the test is passing, and leave it around for 6 hours or so if it is not passing so
it can be debugged. I guess we could even create a ns-limited service account  in each
cluster so developers can easily debug (but without giving them too broad permission).